File: GTTS/INL.TrainingService.Models/TrainingEventInstructor_Item.cs
using System;

namespace INL.TrainingService.Models
{
    public class TrainingEventInstructor_Item : ITrainingEventInstructor_Item
    {
        public long PersonID { get; set; }
        public long TrainingEventID { get; set; }
        public long? PersonsVettingID { get; set; }
        public bool IsTraveling { get; set; }
        public int? DepartureCityID { get; set; }
        public DateTime? DepartureDate { get; set; }
        public DateTime? ReturnDate { get; set; }
        public int? VisaStatusID { get; set; }
        public int? PaperworkStatusID { get; set; }
        public int? TravelDocumentStatusID { get; set; }
        public bool RemovedFromEvent { get; set; }
        public int? RemovalReasonID { get; set; }
        public int? RemovalCauseID { get; set; }
        public DateTime? DateCanceled { get; set; }
        public string Comments { get; set; }
        public int? ModifiedByAppUserID { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/training-event-instructor_item.ts


import { ITrainingEventInstructor_Item } from './itraining-event-instructor_item';

export class TrainingEventInstructor_Item implements ITrainingEventInstructor_Item {
  
	public PersonID: number = 0;
	public TrainingEventID: number = 0;
	public PersonsVettingID?: number;
	public IsTraveling: boolean = false;
	public DepartureCityID?: number;
	public DepartureDate?: Date;
	public ReturnDate?: Date;
	public VisaStatusID?: number;
	public PaperworkStatusID?: number;
	public TravelDocumentStatusID?: number;
	public RemovedFromEvent: boolean = false;
	public RemovalReasonID?: number;
	public RemovalCauseID?: number;
	public DateCanceled?: Date;
	public Comments: string = "";
	public ModifiedByAppUserID?: number;
  
}






